Smart-Table.debug.js

```diff
@@ -608,23 +619,34 @@
                 var index = scope.dataCollection.indexOf(dataRow);
                 if (index !== -1) {
                     selectDataRow(scope.dataCollection, scope.selectionMode, index, dataRow.isSelected !== true);
+                    scope.$emit('selectionChange', {item: dataRow});
```

Comment:
I moved this $emit from the selectDataRow method to here because selectDataRow() is used when selecting multiple or all, and I think we should only $emit once for those cases (rather than for every row changed).